documentation/modules/oauth/con-oauth-authorization-keycloak-authorization-services.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following module:
+//
+// proc-oauth-authorization-broker-config.adoc
+
+[id='con-oauth-authorization-keycloak-authorization-services_{context}']
+= Manage policies and permissions in Keycloak Authorization Services
+
+== Kafka Authorization Model
+
+The link:https://kafka.apache.org/documentation/#security_authz_primitives[Kafka authorization model] defines resource types, and permissions available for each type.
+Whenever some action is performed by the Kafka client, one or more permissions are checked by the broker (depending on the action performed) using the configured authorizer.
+
+There are five resource types: `Topic`, `Group`, `Cluster`, `TransactionalId`, `DelegationToken`.
+
+And these are the different permissions for each resource type:
+
+Topic:
+
+* Create
+* Write
+* Read
+* Delete
+* Describe
+* DescribeConfigs
+* Alter
+* AlterConfigs
+
+Group:
+
+* Read
+* Describe
+* Delete
+
+Cluster:
+
+*  Create
+*  Describe
+*  Alter
+*  DescribeConfigs
+*  AlterConfigs
+*  IdempotentWrite
+*  ClusterAction
+
+TransactionalId:
+
+*  Describe
+*  Write
+
+DelegationToken:
+
+* Describe
+
```

Comment:
additional space

Reply:
WDYM? A space too much and should be removed, or another space needs to be added?

Reply:
Removed one empty line.